### ISSUE TITLE
User consent dialog box timeout too short

### ISSUE THREAD

**Issue by geanferrani123** [Ylianst/MeshCentral#617]
Dear, good afternoon!

Is there any way for me to increase the time user content is open on the user screen? I was informed that when the technician sends the request to connect to the remote desktop, either through the file or through the terminal, the approval screen is for a short time only, would it be possible to enlarge?

The access request screen I am referring to would be this:

![image](https://user-images.githubusercontent.com/55212206/67801480-fa559900-fa67-11e9-91a7-6be671ec5a06.png)

As I said above, this screen for all accesses (desktop, terminal and file) fades very fast, causing the request to time out.

**Comment by krayon007** [comment]
This should be easy, as that value is passed in as a parameter. Should be easy to add a ux on the server to tweak the value it gives the agent.

**Comment by Ylianst** [comment]
The current value is 30 seconds and is set in "meshcore.js" / "meshcore.min.js" in lines like this:

`var pr = require('message-box').create('MeshCentral', this.httprequest.username + ' requesting Terminal Access. Grant access?', 30);`

Let me know what value you want and I can update the server. I could also make it configurable on the server, but I am traveling a lot for the next 3 weeks and so, it would have to be done when I am back.

**Comment by MailYouLater** [comment]
Possibly a duplicate of #547.

Also this timeout was changed from 10 seconds to 30 seconds in this commit: https://github.com/Ylianst/MeshCentral/commit/7e75c7a863b38b747234ca3d576ffd54f6a89461

Also, on a separate note, I really wish you guys would make your commits more 'atomic'. (i.e. one commit does one thing.) The commit I linked above also has a bunch of other stuff in it that's not related to what I'm pointing it out for.
